src/Controls/src/Core/Compatibility/Handlers/Shell/Android/ShellFlyoutTemplatedContentRenderer.cs

```diff
@@ -68,6 +71,14 @@ void OnFlyoutStateChanging(object sender, AndroidX.DrawerLayout.Widget.DrawerLay
 			}
 		}
 
+		class ThingBehavior : AppBarLayout.Behavior
```

Comment:
What exactly is this? And why is it called `ThingBehavior`?

Reply:
Whoops....

That was when I was testing some nested scroll behaviors
Deleted this